mu-plugins/blocks/table-of-contents/postcss/style.pcss

```diff
@@ -1,11 +1,45 @@
-@media (min-width: 890px) {
-	/* stylelint-disable selector-id-pattern */
-	#wp--skip-link--target {
-		scroll-margin-top: var(--wp-local-header-offset, 0);
+.wporg-table-of-contents__header {
+	display: grid;
+	grid-template-columns: 1fr auto;
+	align-items: center;
+
+	& h2 {
+		margin: 0 !important;
+		color: var(--wp--preset--color--charcoal-1);
+		font-family: var(--wp--preset--font-family--inter);
+		font-size: var(--wp--preset--font-size--small);
+		line-height: 1.6;
 	}
 
-	.is-toc-heading {
-		scroll-margin-top: var(--wp-local-header-offset, 0);
+	& button {
+		padding: 0;
+		width: 2rem;
+		height: 2rem;
+		background: transparent;
+		border: none;
+		color: var(--wp--preset--color--blueberry-1);
+
+		&:focus-visible {
+			outline: 1.5px solid var(--wp--preset--color--blueberry-1);
+			outline-offset: -0.5px;
+			box-shadow: none;
+			border-radius: 2px;
+		}
+	}
+}
+
+.wporg-table-of-contents__list {
+	padding-left: 1em;
```

Comment:
```suggestion
	margin-bottom: 0;
	padding-left: 1em;
```

I think we can remove the bottom margin since it's doubled up.

Reply:
I also think we need to increase the line-height to match the designs.

Reply:
> I also think we need to increase the line-height to match the designs.

The designs were updated in [this comment](https://github.com/WordPress/wporg-documentation-2022/pull/58#issuecomment-1516077452). Initially I thought I had the wrong line height so I increased it… but a closer look at the figma, I see there's a 3px margin between each list item. So it should be 1.6 with that margin, and it is now in https://github.com/WordPress/wporg-mu-plugins/pull/389/commits/ecdf69799e84330d0b305ef6bf982c6816904dd9.

<img width="371" alt="Screenshot 2023-05-08 at 3 22 04 PM" src="https://user-images.githubusercontent.com/541093/236913840-0a19da33-b62c-429f-81ab-c7d320e9f568.png">